content/association/_index.md

```diff
@@ -5,37 +5,119 @@ title = "Association"
 > Note:
 >
 > This page is a draft and is subject to change. The association is not yet registered
-> and we are still considering options on how and where it should be. To help us
-> develop these statutes or the association you can join us in the
-> [Biweekly Nordic RSE meeting](/communities/join/#bi-weekly-meetings).
-> 
-> Our suggestion at the moment is to register the association in Finland, since we know the relevant laws.
-> This is not necessarily an informed decision and we would welcome input about the possibility of registering
-> in a different Nordic country.
->
-> It should be possible for members, board members and the chair to not be citizens of the registration country.
-> Other than that, the registration process should be as straightforward as possible.
+> and we are still considering our options.
 >
 
 # The Association
 
-The Nordic RSE group is planning to register in Finland as the Research Software Engineers Ry.
+The Nordic RSE group is planning to register in Finland as the
+Nordic Research Software Engineers Ry.
 Registering as an association comes with certain legal benefits, such as the ability to
 sign contracts and own assets.
 Registering an association will also create a clear membership and governing
 procedures.
 
-The association is governed by an annual assembly of its members, which is held
-mainly online, and its day to day activities are taken care of by a board.
+The association is governed by an annual assembly of it's members, which is held
+mainly online, and it's day to day activities are taken care of by a board.
 
 # Membership
 
-While the purpose of the association is to support research software engineering and
+While purpose of the association is to support research software engineering and
 scientific computing, anyone can join as a member. The only requirement is that you
 wish scientific software was more valued, better implemented, more open and FAIR.
 
-The board of the association will decide annually if a membership fee is collected.
+An annual membership fee is likely to be necessary to cover costs such as the
+domain name and bank account.
+The member's annual meeting will decide the size of the membership fee.
 
 # Statutes
 
-The statutes of the association can be found [here](/association/statutes).
+Below is the proposal for the statutes of the association. While the Finnish
+translation will be official (if the association is registered in Finland,)
+the English translation is equivalent.
```

Comment:
should we double check with an authorized translator that the translations are actually equivalent from a legal perspective?